src/Adapter/Product/QueryHandler/GetEditableCombinationsListHandler.php

```diff
@@ -118,10 +143,15 @@ private function formatCombinationsForEditing(
                 }
             }
 
-            $combinationsForEditing[] = new CombinationForEditing(
+            $impactOnPrice = new DecimalNumber($combination['price']);
+            $combinationsForEditing[] = new EditableCombinationForListing(
                 $combinationId,
                 $this->buildCombinationName($combinationAttributesInformation),
-                $combinationAttributesInformation
+                $combinationAttributesInformation,
+                (bool) $combination['default_on'],
+                $impactOnPrice,
+                $productPrice->plus($impactOnPrice),
+                $this->stockAvailableRepository->getForCombination(new CombinationId($combinationId))->quantity
```

Comment:
```suggestion
                (int) $this->stockAvailableRepository->getForCombination(new CombinationId($combinationId))->quantity
```
 I believe this is what makes behat test fail